docker-builds/e2e/compose.yml

```diff
@@ -0,0 +1,117 @@
+services:
```

Comment:
pls add a readme saying that it runs for 1-2hrs locally and too long at github.
...and also that the tests need maintainance

Reply:
done